core/src/main/resources/AdapterFramework.xsd

```diff
@@ -1,4 +1,136 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified" attributeFormDefault="unqualified">
-	<xsd:element name="IOS-Adaptering" type="xsd:anyType"/>
+	<xsd:element name="configuration" type="ConfigurationType"/>
+	<xsd:element name="IOS-Adaptering" type="ConfigurationType"/>
+	<xsd:element name="module" type="ConfigurationType"/>
```

Comment:
We moeten, nu, gaan afdwingen dat men nu hier het juiste element neerzetten. Ik zie te veel combinaties

- root
- ibis
- module
- IOS-Adaptering

Voordat het helemaal uit de hand loopt, dus een beslissing hier over maken en een grote melding in de console wanneer het fout is.

Reply:
Voor de IbisDoc xsd heb ik destijds een keuze gemaakt. Die volgen? Gaan we op termijn niet tegen de IbisDoc xsd aan valideren? Als mensen hun IDE koppelen aan de IbisDoc xsd zou het niet uit de hand moeten lopen. Is nu ook beschreven in de manual:

https://frank-manual.readthedocs.io/en/latest/gettingStarted/configurationSyntaxChecking.html?highlight=completion#syntax-checking-and-the-frank-doc

Ik zou nog graag een standardisatie doorvoeren over waar we de ibisdoc xsd te zetten en hoe we die noemen want ik zie nu allerlei mensen ze op allerlei plekken zetten. Heb daarvoor Chakir gevraagd dit te doen:

https://github.com/ibissource/iaf/pull/261

Het idee is dat na opstarten deze adapter wordt afgetrapt om de xsd in de configurations folder te zetten en een .gitignore aan te maken zodat die niet gecommit wordt. Ontwikkelaars kunnen dan altijd naar de xsd verwijzen met ../ibisdoc.xsd (of hoe we hem dan ook gaan noemen).